pythonFeed.sh

```diff
@@ -36,8 +39,7 @@ do
 done
 
 if [ "$VERSIONS" != "" ]; then
-    echo "generating cimg-python for versions: $VERSIONS"
-    ./shared/release.sh "$VERSIONS"
+    echo "Created Pull Requests for versions: $VERSIONS"
```

Comment:
if you were to keep the release script here, wouldn't using array expansion e.g ${vers[@]} fix the issue?

Reply:
It would, but I was just trying to keep it quick and since it was already in string format I was just going to keep it as it was

Reply:
you can still keep it in string format. you're just storing it in an array. you would be able to accomplish creating one PR for multiple versions